sagenb/notebook/notebook.py

```diff
@@ -1268,7 +1268,7 @@ def set_not_computing(self):
             W.set_not_computing()
 
     def quit(self):
-        for W in self.__worksheets.values():
+        for W in list(self.__worksheets.values()):
```

Comment:
Not obvious to me what this change was for, but whatever.

Reply:
Thanks. This precise change was because this dictionary is modified inside this loop (some entries are deleted).

Reply:
Yikes.  That's extremely non-obvious from the code.  I'd probably refactor if I cared more but since we're just in legacy maintenance mode I'll leave it.